templates/StatefulSet-zabbix-server.yaml

```diff
@@ -24,6 +24,24 @@ spec:
         app.kubernetes.io/instance: {{ .Release.Name }}-zabbix-server
         app.kubernetes.io/managed-by: {{ .Release.Service }}-zabbix-server
     spec:
+      nodeSelector:
```

Comment:
Hi @adamcharnock!

Could you remove the lines 27 to 32?
The goal is to leave only the lines 33 to 44 that will be filled in only when someone sets the custom values for these properties.